staging/src/k8s.io/csi-translation-lib/translate.go

```diff
@@ -35,9 +35,36 @@ var (
 	}
 )
 
+// CSITranslator translates in-tree storage API objects to their equivilant CSI
+// API objects. It also provides many helper functions to determine whether
+// translation logic exists and the mappings between "in-tree plugin <-> csi driver"
+type CSITranslator interface {
```

Comment:
This looks great! I would say that, in a package with "CSI" in the name, you could even call this type `Translator`. I won't insist on it though.